app/views/internal/configs/show.html.erb

```diff
@@ -384,7 +384,7 @@
   </div>
 </div>
 
-<% if current_user.has_role?(:single_resource_admin, Config) %>
+<% if current_user.has_role?(:single_resource_admin, Config) && current_user.has_role?(:super_admin) %>
```

Comment:
Is `if current_user.has_role?(:single_resource_admin, Config)` necessary? I believe that a super admin will pass this check no matter what and a non-super admin would fail the second check.